src/Wallabag/CommentBundle/Entity/Comment.php

```diff
@@ -0,0 +1,287 @@
+<?php
+
+namespace Wallabag\CommentBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation\ExclusionPolicy;
+use JMS\Serializer\Annotation\Exclude;
+use JMS\Serializer\Annotation\VirtualProperty;
+use JMS\Serializer\Annotation\SerializedName;
+use Wallabag\UserBundle\Entity\User;
+use Wallabag\CoreBundle\Entity\Entry;
+
+/**
+ * Comment.
+ *
+ * @ORM\Table(name="comment")
+ * @ORM\Entity(repositoryClass="Wallabag\CommentBundle\Repository\CommentRepository")
+ *
+ * @ExclusionPolicy("none")
+ */
+class Comment
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="text", type="text")
+     */
+    private $text;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="created", type="datetime")
+     */
+    private $created;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="updated", type="datetime", nullable=true)
+     */
+    private $updated;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="quote", type="string", length=255, nullable=true)
+     */
+    private $quote;
+
+    /**
+     * @var array
+     *
+     * @ORM\Column(name="ranges", type="array", length=255, nullable=true)
+     */
+    private $ranges;
+
+    /**
+     * @Exclude
+     *
+     * @ORM\ManyToOne(targetEntity="Wallabag\UserBundle\Entity\User")
+     */
+    private $user;
+
+    /**
+     * @Exclude
+     *
+     * @ORM\ManyToOne(targetEntity="Wallabag\CoreBundle\Entity\Entry", inversedBy="comments")
+     * @ORM\JoinColumn(name="entry_id", referencedColumnName="id")
+     */
+    private $entry;
+
+    /*
+     * @param User     $user
+     */
+    public function __construct(\Wallabag\UserBundle\Entity\User $user)
+    {
+        $this->user = $user;
+        $this->created = new \DateTime();
+    }
+
+    /**
+     * Get id.
+     *
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set text.
+     *
+     * @param string $text
+     *
+     * @return Comment
+     */
+    public function setText($text)
+    {
+        $this->text = $text;
+
+        return $this;
+    }
+
+    /**
+     * Get text.
+     *
+     * @return string
+     */
+    public function getText()
+    {
+        return $this->text;
+    }
+
+    /**
+     * Set created.
+     *
+     * @param \DateTime $created
+     *
+     * @return Comment
+     */
+    public function setCreated($created)
```

Comment:
Did you plan to use these methods?
I think you should better handle created / update the way we did in [Entry](https://github.com/wallabag/wallabag/blob/v2/src/Wallabag/CoreBundle/Entity/Entry.php#L355-L366).